integration/container/nat_test.go

```diff
@@ -56,6 +56,7 @@ func TestNetworkLocalhostTCPNat(t *testing.T) {
 }
 
 func TestNetworkLoopbackNat(t *testing.T) {
+	skip.If(t, testEnv.GitHubActions) // https://github.com/moby/moby/issues/41561
```

Comment:
Can you add a "FIXME" comment in the skip itself?

```suggestion
	skip.If(t, testEnv.GitHubActions, "FIXME: Flaky test; see https://github.com/moby/moby/issues/41561" )
```

Reply:
^^ wow, interesting; looks like a bug on GitHub; this is the markdown source:

<img width="742" alt="Screenshot 2022-08-30 at 11 29 42" src="https://user-images.githubusercontent.com/1804568/187402347-65651c9e-50fa-4578-8599-573a871aa080.png">

And this is what it shows: (notice `moby/moby` being stripped :warning:)

<img width="753" alt="Screenshot 2022-08-30 at 11 29 32" src="https://user-images.githubusercontent.com/1804568/187402353-0b9b70e2-0bb7-40ab-89be-0b946f94d1d5.png">